### ISSUE TITLE
No output

### ISSUE THREAD

**Issue by bebbi** [neo4j-graphql/neo4j-graphql-js#36]
I can see cypher queries corresponding to the graphql endpoints being logged, everything appears to work (auth, bolt connection, all seems to work, i.e. does appear to complain if set differently), and I get a 200 return with expected formatting in the log:

```
[200] {"statusCode":200,"headers":{"Content-Type":"application/json","Access-Control-Allow-Origin":"*"},"body":"{\"data\":{\"allItems\":[]}}"}

```
But all the results, whether lists or single items, are always empty (either `null` or `[]` depending on expected type)

I'm a bit stumped by this behaviour. How can I troubleshoot it?

This is using a docker image with neo4j:3.3.4, apoc-3.3.0.2-all.jar, (and coincidentally, neo4j-graphql-3.3.0.1.jar). Calling neo4j-graphql-js from an ephemeral server created by `apollo-server-lambda`.

Also, copy-pasting the logged queries into the browser console at :7474 does return the expected values.

**Comment by johnymontana** [comment]
As a starting point, if you copy/paste the Cypher queries being logged and run them in Neo4j Browser do you see the expected results from Neo4j Browser?

**Comment by bebbi** [comment]
Good timing :) I had just edited the post to add that (at the bottom)

**Comment by bebbi** [comment]
Some more troubleshooting:

1. Verify it's not interference with another db (done)
Stop docker container -> `GraphQL error: connect ECONNREFUSED 127.0.0.1:7687`
2. Verify same behaviour when using neo4j:3.2 (done)
3. Add more logging to neo4j (failed)
Looks like query logging is only available in [enterprise](https://neo4j.com/docs/operations-manual/current/monitoring/logging/query-logging/) so not helpful here.

What options do we have to debug neo4j and check whether input/output is as expected?

**Comment by bebbi** [comment]
This is at least partially solved:
For some reason, the neo4j browser console didn't accept "paste" when I copy/pasted the query manually, I forgot to add the labels - the query assumes that nodes of graphql types are labeled, e.g.:

    MATCH (item:Item {}) ...

However, the nodes were created with `neo4j-graphql` (and same graphql schema) which as it appears doesn't label nodes with type names.

At this stage, I'd like try playing with creating nodes but it looks like this is blocked by #35

**Comment by bebbi** [comment]
edit: previous issue updated

**Comment by bebbi** [comment]
Closing this as probably duplicate of #22.
